test/purchases_flutter_test.dart

```diff
@@ -82,4 +82,22 @@ void main() {
         ['monthly_intro_pricing_one_week']);
     expect(list.length, 1);
   });
+
+  test('canMakePayments with no params calls successfully', () async {
+    response = true;
+    await Purchases.canMakePayments();
+    expect(log, <Matcher>[
+      isMethodCall('canMakePayments', arguments: {'features': []})
+    ]);
```

Comment:
🤔 
I'd also check that the value is passed along correctly, and have the response be random so that if for some reason we make a change that makes the code only compatible with `true`, it gets caught: 

```dart
response = randomGenerator.nextBool();
var receivedCanMakePayments = await Purchases.canMakePayments();
expect(receivedCanMakePayments, response);
```
that way if we later broke [this line](https://github.com/RevenueCat/purchases-flutter/pull/193/files#diff-50113335b625b38d77e98819f82d0437574295a701bcabd0450f59d5f7f12507R522
) tests would fail

Reply:
i passed a parameter in the second test, were you thinking of something else?

good call on the random 👍